libraries/libfc/CMakeLists.txt

```diff
@@ -73,7 +73,7 @@ set( fc_sources
      src/network/udp_socket.cpp
      src/network/url.cpp
      src/network/http/http_client.cpp
-     src/compress/smaz.cpp
+#     src/compress/smaz.cpp
```

Comment:
No need to comment all these out -- just remove them (and a couple of the others that are already commented out too)